wlroots 0.13.0 is out-of-date and has no other consumers. Keep it
private to gamescope package for now.

--- subprojects/wlroots/include/meson.build.orig	2021-04-07 19:19:31 UTC
+++ subprojects/wlroots/include/meson.build
@@ -17,7 +17,9 @@ if not features.get('xdg-foreign')
 	]
 endif
 
+if not meson.is_subproject()
 install_subdir('wlr',
 	install_dir: get_option('includedir'),
 	exclude_files: exclude_files,
 )
+endif
--- subprojects/wlroots/include/wlr/meson.build.orig	2021-04-07 19:19:31 UTC
+++ subprojects/wlroots/include/wlr/meson.build
@@ -21,4 +21,6 @@ ver_h = configure_file(
 	configuration: version_data,
 )
 
+if not meson.is_subproject()
 install_headers(conf_h, ver_h, subdir: 'wlr')
+endif
--- subprojects/wlroots/meson.build.orig	2021-04-07 19:19:31 UTC
+++ subprojects/wlroots/meson.build
@@ -153,7 +153,7 @@ lib_wlr = library(
 	soversion: soversion,
 	dependencies: wlr_deps,
 	include_directories: [wlr_inc, proto_inc],
-	install: true,
+	install: not meson.is_subproject(),
 	link_args: symbols_flag,
 	link_depends: symbols_file,
 )
